src/content/docs/data-apis/manage-data/drop-data-using-nerdgraph.mdx

```diff
@@ -36,7 +36,7 @@ Learn more about dropping data in this video (7:09 minutes):
 
 Besides creating drop-data rules, other ways to minimize unwanted data include:
 
-* If you want to drop APM metric timeslice data, you can use [metric normalization rules](/docs/new-relic-one/use-new-relic-one/ui-data/metric-normalization-rules).
+* If you want to drop [APM metric timeslice data](/docs/data-apis/understand-data/new-relic-data-types/#timeslice-data) (any metric queried with `newrelic.timeslice.value` or a name beginning with `apm.*`), you can use [metric normalization rules](/docs/new-relic-one/use-new-relic-one/ui-data/metric-normalization-rules). You cannot drop metric timeslice data with drop rules.
```

Comment:
I don't think it's necessary to provide this level of detail on identifying metric timeslice data in the parentheses. Personally, I think it's best if we just link out to this document on querying metric timeslice data with NRQL:

https://docs.newrelic.com/docs/data-apis/understand-data/metric-data/query-apm-metric-timeslice-data-nrql/

This will be safer, too, if the paradigm for querying metric timeslice data ever changes.

So that being said, I would suggest:

* Removing the statement in parentheses
* Replacing the **APM metric timeslice data** link with the above link instead

Reply:
I kind of split the difference between your suggestion and mine. I think the link I provided was more helpful to understanding the important details about what the two types of metrics are, which is basic to using drop rule, your doc I found very informative but felt more geared towards advanced users who already know about metrics instead of a just a clear definition like: _X works, Y doesn't_

I removed the parens as suggested.

I'm interested in the less-technical doc team's opinion on what they find more helpful to clarify the concept that `Metric` data types is really two types of data: dimensional metrics which can be dropped, and timeslice metrics which can't